lib/gen_stage.ex

```diff
@@ -1103,6 +1116,27 @@ defmodule GenStage do
   @callback handle_info(message :: term, state :: term) ::
               {:noreply, [event], new_state}
               | {:noreply, [event], new_state, :hibernate}
+              | {:noreply, [event], new_state, {:continue, term}}
+              | {:stop, reason :: term, new_state}
+            when new_state: term, event: term
+
+  @doc """
+  Invoked to handle `continue` instructions.
+
+  It is useful for performing work after initialization or for splitting the work
+  in a callback in multiple steps, updating the process state along the way.
```

Comment:
I think this might be a bit misleading. I think the biggest use of this is being able to emit events and then do something right after, being sure that no messages, calls, or anything else will be "injected" in between. Maybe we can rephrase this?

My suggestion:

```suggestion
  This callback can be used to perform work right after emitting events from
  other callbacks. The "continue mechanism" makes sure that no messages,
  calls, casts, or anything else will be handled between a callback emitting
  a `:continue` tuple and the `c:handle_continue/2` callback being invoked.
```